python/packages/ai/teams/ai/planner/command_type.py

```diff
@@ -5,6 +5,6 @@
 from enum import Enum
 
 
-class CommandType(Enum):
+class CommandType(str, Enum):
```

Comment:
Can you share why we need to inherit `str`?

Reply:
enums in python default to being `unknown`, if you want the enum to be of type string you need to also inherit from str

Reply:
Thank you for the explanation. I'm curious why we want to make it to be type string? When we write `CommandType.Do`, do we need to care what's the type of the underlying value?

Reply:
Ya its just good to know if an enum is a `string` enum or a `int` enum at least from an intellisense perspective. Not necessarily _needed_ since typing in python is optional anyway, but I personally like the types to be as accurate as possible especially in cases like these that don't require alot of work.